scripts/traffle.lic

```diff
@@ -82,326 +84,365 @@ module Traffle
+      raffle_data[2] = [0, "test description 2","[Town Crier Public Headquarters]",Time.now,"1337",5,"-1","gambling floor"]
+      raffle_data[3] = [0, "test description 3","[Town Crier Secret Headquarters]",Time.now,"1337",5,"-2","gambling table"]
+      raffle_data[4] = [0, "test description 4","[Town Crier Public Headquarters]",Time.now,"1337",5,"-1","gambling floor"]
+      raffle_data[5] = [0, "test description 5","[Town Crier Secret Headquarters]",Time.now,"1337",5,"-2","gambling table"]
+      raffle_data[6] = [0, "test description 6","[Town Crier Public Headquarters]",Time.now,"1337",5,"-1","gambling floor"]
+      raffle_data[7] = [0, "test description 7","[Town Crier Secret Headquarters]",Time.now,"1337",5,"-2","gambling table"]
+      raffle_data[8] = [0, "test description 8","[Town Crier Public Headquarters]",Time.now,"1337",5,"-1","gambling floor"]
+      raffle_data[9] = [0, "test description 9","[Town Crier Secret Headquarters]",Time.now,"1337",5,"-2","gambling table"]
+  end
+  response = nil
+  raffle_data.each { |raf| 
+    _respond "
+    TownCrier_API.post_raffle(
+        #{raf[3]},
+      \"#{raf[1].to_s.gsub(/"/, "&quot;")}, Number of Winners: #{raf[6].to_s}\",
+      \"#{raf[5].to_s.reverse.gsub(/(\d{3})(?=\d)/, '\\1,').reverse} silvers\",
+       \"Located On: #{raf[7].to_s}, Room Name: #{raf[2].to_s}\",
+       #{raf[4].to_s},
+       \"#{XMLData.name}\"
+       )
+    " if @debug
+    response = TownCrier_API.post_raffle(
+        raf[3],
+      "#{raf[1].to_s.gsub(/"/, "&quot;")}, Number of Winners: #{raf[6].to_s}",
+        "#{raf[5].to_s.reverse.gsub(/(\d{3})(?=\d)/, '\\1,').reverse} silvers",
+       "Located On: #{raf[7].to_s}, Room Name: #{raf[2].to_s}",
+       raf[4].to_s,
+       "#{XMLData.name}"
+       )
+      _respond "response code: #{response.code}" if response.code.to_i != 200;
+      _respond "response body: #{response.body}" if response.body =~ /error/i;
+      response = nil
+    }
+
+  exit
+=begin
+  if raffle_data[0]
+    raffle_csv = File.open("#{$script_dir}raffle.tsv", "w")
+    raffle_csv.puts("Date\tTime\tDescription\tCost\tLocation\tRoomNum")
+    raffle_data.each{ |raf| 
+      output = []
+      output << raf[3].strftime("%Y-%m-%d").to_s #date
+      output << raf[3].strftime("%I:%M %p").to_s #time
+      output << "#{raf[1].to_s.gsub(/"/, "&quot;")}, Number of Winners: #{raf[6].to_s}" #ticket details
+      output << "#{raf[5].to_s.reverse.gsub(/(\d{3})(?=\d)/, '\\1,').reverse} silvers" #cost
+      output << "Located On: #{raf[7].to_s}, Room Name: #{raf[2].to_s}" #table name, #room_name
+      output << raf[4].to_s #lich#
+      add_line = output.join("\t")
+      raffle_csv.puts add_line
+    }
+    raffle_csv.close
+    respond "#{$script_dir}raffle.tsv created!"
+  else
+    _respond "No raffles being tracked.  Run this script while at the raffle table!"
+    _respond "Completed raffles are removed after #{timelimit} minutes."
+  end
+=end

```

Comment:
We can probably just delete this commented out section between the `=begin` and `=end`. If we need to get the code back, we can grab it from an older version in git.